lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -40,7 +40,14 @@ defmodule Membrane.H264.FFmpeg.Encoder do
                :placebo
              ]
 
-  def_options crf: [
+  def_options add_dts?: [
+                spec: boolean(),
+                default: false,
+                description: """
+                Setting this flag to true causes decoder to add presentation timestamp (dts) taken from buffer timestamp into the AVFrame and in consequence to the produced packet.
+                """
+              ],
```

Comment:
1. dts is not presentation ts
2. same as above, I don't see the point of not passing timestamps